src/bug_report.rs

```diff
@@ -30,10 +36,17 @@ pub fn create() {
             .unwrap_or(link);
 
         println!(
-            "Click this link to create a GitHub issue populated with your configuration:\n\n  {}",
+            "Your environment info has been copied to your clipboard. Click this link to create a GitHub issue:\n\n  {}",
             link
         );
     }
+
+    if copy_success {
```

Comment:
This reads as "if copying into the clipboard was successful, print 'Your clipboard was unavailable'". Shouldn't this be the other way around? (Also see `is_err`)

Reply:
I just noticed this too...